conda_build/build.py

```diff
@@ -322,7 +322,7 @@ def build(m, get_src=True, verbose=True, post=None):
 
         # Display the name only
         # Version number could be missing due to dependency on source info.
-        print("BUILD START:", m.dist())
+        print("BUILD START:", m.name())
```

Comment:
What @tswicegood said. Also, the comment on the lines above the change address the issue.